elements/lisk-p2p/test/unit/peer_pool.ts

```diff
@@ -1006,7 +1007,49 @@ describe('peerPool', () => {
 		});
 	});
 
-	describe.skip('#_evictPeer', () => {});
+	describe('#_evictPeer', () => {
+		const whitelistedPeers = [
+			{ peerId: '1.2.3.4:5000', ipAddress: '1.2.3.4', wsPort: 5000 },
+		];
+		const fixedPeers = [
+			{ peerId: '5.6.7.8:5000', ipAddress: '5.6.7.8', wsPort: 5000 },
+		];
+		const defaultPeers = [
+			{
+				id: '69.123.456.78:5000',
+				peerId: '69.123.456.78:5000',
+				ipAddress: '69.123.456.78',
+				wsPort: 5000,
+			},
+			...whitelistedPeers.map(peer => ({ ...peer, id: peer.peerId })),
+			...fixedPeers.map(peer => ({ ...peer, id: peer.peerId })),
+		];
+
+		beforeEach(async () => {
+			(peerPool as any)._peerLists.whitelisted = whitelistedPeers;
+			(peerPool as any)._peerLists.fixedPeers = fixedPeers;
+			sandbox.stub(peerPool as any, 'getPeers').returns(defaultPeers);
+			sandbox.stub(peerPool, 'removePeer');
+		});
+
+		it('should not evict whitelisted peer', async () => {
```

Comment:
Can we add another case to check a positive eviction?

Reply:
Added positive case too